scopes/dns/variables.tf

```diff
@@ -0,0 +1,11 @@
+variable "project_id" {
```

Comment:
noch referenzieren in google_dns_managed_zone?

Reply:
Muss man nicht. Kommt implizit via https://github.com/senacor/lpt-bootstrap/pull/6/files#diff-1c96dd0aecb823173ffde55378625b877428b7a6cbeaf492fa7f166e140377deR16